peripherals/wwdg/wwdg.yaml

```diff
@@ -0,0 +1,38 @@
+# The most common features, found on Basic timers TIM6 and TIM7,
+# and found on all more advanced timers.
+
+WWDG:
+  CR:
+    _modify:
+      WDGA: 
+        description: "Activation bit for the watchdog. This bit is set by software and only cleared after a reset."
+        bitOffset: 7
+        bitWidth: 1
+      T:
+        description: "These bits contain the value of the watchdog counter. A reset is produced when it rolls over from 0x40 to 0x3F (T6 us cleared)."
+        bitOffset: 0
+        bitWidth: 7
+  CFR:
+    WDGTB:
+      Div1: [0, "Counter clock (PCLK1 div 4096) div 1"]
+      Div2: [1, "Counter clock (PCLK1 div 4096) div 2"]
+      Div4: [2, "Counter clock (PCLK1 div 4096) div 4"]
+      Div8: [3, "Counter clock (PCLK1 div 4096) div 8"]
+      _merge:
+        - "WDGTB*"
+    _modify:
+      EWI:
+        description: "Used to enable early wakeup interrupt. This allows an interrupt to be serviced before device reset. Only cleared after a reset"
+        bitOffset: 9
+        bitWidth: 1
+      W:
+        description: "Window value to be compared to the downcounter."
+        bitOffset: 0
+        bitWidth: 7
```

Comment:
As with the WDGA and T fields, I think EWI and W should already exist in the SVD files, so you just need to give them variants and/or ranges here. Since EWI is just "set" or "reset" it doesn't really need enumerated values, but W would benefit from being told it can take any value [0, 127] (so that the setter functions become safe).